osac-operator/internal/dummy/cleanapi/cleanapi.go

```
@@ -0,0 +1,18 @@
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+language governing permissions and limitations under the License.
+*/
+
+// This package and this file only exist because there is no way to tell the protobuf Go code generator that we are not
+// generating code for the 'cleanapi' module. We only import that module to use its annotations, but the Go code generator
+// will anyhow include an import statement for the package. So keep this, even if it is empty.
+
+package cleanapi
```

golangci-lint (2.12.2)

[error] ST1000: at least one file in a package should have a package comment

(staticcheck)